src/test/java/uk/gov/hmcts/reform/bsp/common/model/document/CtscContactDetailsTest.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.bsp.common.model.document;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class CtscContactDetailsTest {
+
+    private static final String CTSC_CARE_OF = "c/o HMCTS Digital Divorce";
```

Comment:
I've commented this in one of John's PRs. I think the "c/o " bit should be in Docmosis, not here.
I've seen other templates that use this information, but without the "c/o"

Reply:
it's a mocked data for testing :)